sumpy/kernel.py

```diff
@@ -262,7 +262,7 @@ def get_code_transformer(self) -> Callable[[Expression], Expression]:
         return lambda expr: expr
 
     @abstractmethod
-    def get_expression(self, dist_vec: sp.Matrix) -> ArithmeticExpr:
```

Comment:
`ArithmeticExpr` can probably go here too? There are different aliases for it in `kernel` and `symbolic` too, which is not great :(